utils/static-executable-args.lnk

```diff
@@ -13,3 +13,4 @@
 -licuuc
 -licudata
 -ldl
+-lstdc++
```

Comment:
I think the failures in `static-stdlib-linux.swift` are pointing that there's a missing `-lm` here.

There seems to also fail at a couple of C++ symbols, so there might be something else wrong (maybe one needs to use `clang++` for static linking?).

Reply:
Yeah, I had the change for the `-lm` already, I missed the permissions change (stupid Windows).